src/spectrecoff-cli/commands/Layout.fs

```diff
@@ -20,12 +20,12 @@ type LayoutExample() =
 
         let canvasContent =
             canvas (Width 12) (Height 12)
-            |> withPixels (Rectangle (Point(0,0), Point(11,11))) Color.Yellow
-            |> withPixels (Rectangle (Point(2,2), Point(4,3))) Color.Purple
-            |> withPixels (Rectangle (Point(7,2), Point(9,3))) Color.Purple
-            |> withPixels (ColumnSegment (ColumnIndex 8, StartIndex 4, EndIndex 7)) Color.Blue
-            |> withPixels (RowSegment (RowIndex 9, StartIndex 3, EndIndex 8)) Color.Purple
-            |> withPixels (Pixels [(3,10); (8,10)]) Color.Purple
+            |> withPixels (Rectangle(Point(0, 0), Point(11, 11))) Color.Yellow
```

Comment:
I don't agree with removing the space between `Rectangle` and `(...)`. The () are not method parameter braces, but belong to the tuple definition, which is an argument to the Rectangle case constructor. And in the same way you'd write `add (x, y)` for and add function that takes a tuple, this should be `Rectangle (Point (0, 0), Point (11, 11))`

Reply:
I am actually surprised that it compiles, because `let six = addTwo4` would not.

Reply:
Removed the spaces, but left the ones inside the tuples :)